test/playground.html

```diff
@@ -71,7 +71,7 @@
 
       autocomplete('#contacts2', {
         debug: true,
-        keyboardShortcuts: [191, 83],
+        keyboardShortcuts: [191, 's'],
```

Comment:
Nevermind, if you want a key that is not a character, you need keycodes (like showing the search when `SPACE` is used)